TATB-01 : Make login and registration page responsive till tablet view

Ishajtdev opened the pull request:
-made login and registration page responsive

DipeshGandhi9 committed:
Innitial Comment

Ishajtdev committed:
TATB-01 :
  -made login and registration page responsive

Ishajtdev committed:
TATB-01 :
  -made login and registration page responsive

Ishajtdev committed:
TATB-01 :
  -made login and registration page responsive

PoojanPatelJT commented on src/app/authorization/login/login.component.html:
```diff
@@ -1,60 +1,84 @@
-    </div>
-    
-</div>
+<div class="row align-items-center" style="height: 100vh">
```
please move inline css to external one

PrabhatJTDeveloper replied:
yes, i removed it.

PrabhatJTDeveloper replied:
yes, i moved it to the external css.

PoojanPatelJT commented on src/app/authorization/login/login.component.html:
```diff
@@ -1,60 +1,84 @@
+        <div class="col-md-12">
+          <div class="text-center">
+            <label>
+              Don't Have an Account ?<a href="/registration"> Register Now</a>
```
Please use routerLink instead of href

PrabhatJTDeveloper replied:
i changed href to the routerlink.

PrabhatJTDeveloper replied:
yes, i changed the href to routerlink

PoojanPatelJT commented on src/app/authorization/login/login.component.ts:
```diff
@@ -1,11 +1,16 @@
 import { FormBuilder, FormControl, FormGroup, NgModel, Validators } from '@angular/forms';
 import { Router } from '@angular/router';
-import { QuestionService } from '../../service/question.service';
+
```
Do code intention

PrabhatJTDeveloper replied:
i done code intention

PrabhatJTDeveloper replied:
yes, i done it

PoojanPatelJT commented on src/app/authorization/login/login.component.ts:
```diff
@@ -51,6 +57,8 @@ export class LoginComponent implements OnInit, OnDestroy {
+    this.initForm();
     if (!localStorage.getItem('registeruser')?.length) {
       let registeruser: any = localStorage.getItem('registeruser');
       registeruser = JSON.parse(registeruser as string);
```
No need to declare as string

PrabhatJTDeveloper replied:
done the changes

PrabhatJTDeveloper replied:
okay . i removed it

PoojanPatelJT commented on src/app/authorization/login/login.component.ts:
```diff
@@ -60,17 +68,41 @@ export class LoginComponent implements OnInit, OnDestroy {
+    this.route.navigateByUrl('/login');
+  }
+
+  initForm() {
```
Why there is two function that is creating form first one is initForm and createForm() please remove this

PrabhatJTDeveloper replied:
removed initForm() function and done all functionality in createForm() function

PrabhatJTDeveloper replied:
okay . i removed initForm function and make all the functionality in the createForm function.

PoojanPatelJT commented on src/app/authorization/login/login.component.ts:
```diff
@@ -60,17 +68,41 @@ export class LoginComponent implements OnInit, OnDestroy {
 
-
+  get adminFormValidator(){
+   console.log(this.adminForm);
```
Please remove console from here

PrabhatJTDeveloper replied:
removed

PrabhatJTDeveloper replied:
okay, i removed

PoojanPatelJT commented on src/app/authorization/registration/registration.component.html:
```diff
@@ -1,29 +1,26 @@
-<div class="card shadow" style="width: 35rem; margin-top: 5rem; margin-bottom: 4rem;">
-    <div class="card-body" style="padding: 0px;">
-<form [formGroup]="registrationForm" (ngSubmit)="submitform()" class="p-3">
+<div class="row align-items-center" style="height: 100vh;">
```
Please remove inline css from here

PrabhatJTDeveloper replied:
yes, i removed inline css

PrabhatJTDeveloper replied:
yes, i removed it.

PoojanPatelJT commented:
Please wait till https://github.com/Jyoti-Technosoft/technical-assistant/pull/4 this PR need to merge first as there is many changes

Ishajtdev committed:
TATB - remove inline css and use form builder instead of form control.

Ishajtdev committed:
TATB - remove inline css and use form builder instead of form control.

Ishajtdev committed:
TATB - remove inline css and use form builder instead of form control.

Ishajtdev committed:
TATB - make change in css file of login & registration page

Ishajtdev committed:
TATB - make change in css file of login & registration page

Ishajtdev committed:
TATB - make change in css file of login & registration page

Ishajtdev committed:
TATB - make change in css file of login & registration page

PoojanPatelJT commented on src/app/app.module.ts:
```diff
@@ -1,36 +1,31 @@
-import { AppRoutingModule } from './app-routing.module';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { HttpClientModule } from '@angular/common/http';
+import '@angular/localize';
```
Could you Please remove this ? and     "@angular/localize/init", add this in your angular.json file in polyfills

PrabhatJTDeveloper replied:
yes, removed it and added to the angular.json

PoojanPatelJT commented on src/app/app.module.ts:
```diff
@@ -1,36 +1,31 @@
 import { BrowserModule } from '@angular/platform-browser';
-import { AppRoutingModule } from './app-routing.module';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { HttpClientModule } from '@angular/common/http';
```
We also no need for http module as there is no api calls

PrabhatJTDeveloper replied:
okay, so i removed it.

PoojanPatelJT commented on src/app/authorization/login/login.component.css:
```diff
@@ -1,13 +1,17 @@
-    height: auto;    
-}
\ No newline at end of file
+* {
```
why do we need to apply roboto font to all css ? is there any requirement

PrabhatJTDeveloper replied:
yes need same font in whole project so need it but not in all css file so i changed it and give the font to the style.css file.

PoojanPatelJT commented on src/app/authorization/login/login.component.css:
```diff
@@ -1,13 +1,17 @@
+}
+
+.form-text {
+  font-size: calc(70% + 1vh);
```
here calc will not work as ther is no scss file so calculation will be failed could you please check

PrabhatJTDeveloper replied:
yes, i checked it and i changed the file from css to scss.

PoojanPatelJT commented on src/app/authorization/login/login.component.css:
```diff
@@ -1,13 +1,17 @@
+}
+
+.form-control {
+  font-size: calc(70% + 1vh);
```
here same scss file

PrabhatJTDeveloper replied:
yes, i changed it

PoojanPatelJT commented on src/app/authorization/login/login.component.html:
```diff
@@ -1,60 +1,92 @@
+        class="rounded-4 mt-5 mb-5 bg-light"
+      >
+        <div class="col text-center p-3">
+          <img src="assets/img/login.png" width="60px" height="60px" />
```
instad of giving width height here could you please change to external css

PrabhatJTDeveloper replied:
yes , i changed to external css

PoojanPatelJT commented on src/app/authorization/registration/registration.component.css:
```diff
@@ -1,33 +1,16 @@
-*{
-    font-family: Roboto;
+* {
+  font-family: Roboto;
```
Why giving all font family to roboto ?

PrabhatJTDeveloper replied:
because want same font family to the whole project

PoojanPatelJT commented on src/app/app-routing.module.ts:
```diff
@@ -1,22 +1,18 @@
 
 const routes: Routes = [
   {path: '', redirectTo:'/login', pathMatch:'full' },
   {path: 'login', component:LoginComponent ,canActivate:[AuthGuard]},
```
Please give proper indentation

PrabhatJTDeveloper replied:
yes, i changed it

PoojanPatelJT commented on src/app/authorization/login/login.component.html:
```diff
@@ -1,60 +1,92 @@
+            <div
+              class="form-text text-danger"
+              *ngIf=" adminFormValidator['emailId']?.errors?.['email'] && (adminFormValidator['emailId']?.dirty || adminFormValidator['emailId']?.touched)"
+            >
```
please wrap if condition properly inside () bracket.

PrabhatJTDeveloper replied:
yes, i changed it.

PoojanPatelJT commented on src/app/authorization/registration/registration.component.html:
```diff
@@ -1,129 +1,241 @@
+            <div
+              class="form-text text-danger"
+              *ngIf="FullName.errors?.['required'] && (FullName.touched || FullName.dirty)"
+            >
```
Please use camelCase here for FullName Because we are not using Pascal case

PrabhatJTDeveloper replied:
this is already changed in https://github.com/Jyoti-Technosoft/technical-assistant/pull/4 PR.

Ishajtdev committed:
- TATB-01
  - remove the unnecessarythings and change the html,scss,style.css,angular.json etc.

Ishajtdev committed:
- TATB-01
  - remove the unnecessarythings and change the html,scss,style.css,angular.json etc.

Ishajtdev committed:
- TATB-01
  - remove the unnecessarythings and change the html,scss,style.css,angular.json etc.

Ishajtdev committed:
- TATB-01
  - again change the angular.json and app.module.ts file

Ishajtdev committed:
- TATB-01
  - again change the angular.json and app.module.ts file

Ishajtdev committed:
- TATB-01
  - again change the angular.json and app.module.ts file

Ishajtdev committed:
TATB:01 :
   -changed css type
   - removed css file
   - cleanup question service file
    -changed import in app service and removed http client module

Ishajtdev committed:
TATB:01 :
   -changed css type
   - removed css file
   - cleanup question service file
    -changed import in app service and removed http client module

Ishajtdev committed:
TATB:01 :
   -changed css type
   - removed css file
   - cleanup question service file
    -changed import in app service and removed http client module

charmijtdev reviewed (approved):
Changes Are done Approved

PoojanPatelJT commented:
Approved

PoojanPatelJT committed:
Reverting changes and making develop branch up to date  (#18)

* -revert Chagnes

* -revert changes to scss

* -added angualr file

* -changes css type

PoojanPatelJT committed:
Bug/revert changes (#19)

* -revert Chagnes

* -revert changes to scss

* -added angualr file

* -changes css type

* -changes

* changes revert for gitignore

PoojanPatelJT committed:
git igonre

PoojanPatelJT committed:
git develop

PoojanPatelJT committed:
Merge branch 'develop' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/TATB-01-BUG-make-login-registration-page-responsive

PoojanPatelJT committed:
-added file change

PoojanPatelJT committed:
-added file change

charmijtdev commented:
Please create NEw

PoojanPatelJT committed:
Merge branch 'feature/TATB-01-BUG-make-login-registration-page-responsive' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/TATB-01-BUG-make-login-registration-page-responsive

PoojanPatelJT committed:
-added removed code

PoojanPatelJT committed:
-added removed code

PoojanPatelJT committed:
Merge branch 'feature/TATB-01-BUG-make-login-registration-page-responsive' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/TATB-01-BUG-make-login-registration-page-responsive

PoojanPatelJT committed:
-login page

PoojanPatelJT commented:
Not merge to develop

charmijtdev committed:
Merge branch 'develop' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/TATB-01-BUG-make-login-registration-page-responsive

charmijtdev commented:
Creating new PR

charmijtdev committed:
Revert "Merge branch 'develop' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/TATB-01-BUG-make-login-registration-page-responsive"

This reverts commit d4c72b303789cc6dde8b63294a79a5067e648c12.

charmijtdev committed:
Revert "Merge branch 'develop' of https://github.com/Jyoti-Technosoft/technical-assistant into feature/TATB-01-BUG-make-login-registration-page-responsive"

This reverts commit d4c72b303789cc6dde8b63294a79a5067e648c12.

charmijtdev committed:
Revert "-login page"

This reverts commit d42d5cfb628bfbc245ad22dfd10e8b93697335be.

charmijtdev commented:
Creating New PR
